jumpstart.sh

```diff
@@ -26,13 +27,16 @@ case $argument in
         print_usage
         ;;
     -s|--high-sierra)
-        "$TOOLS/FetchMacOS/fetch.sh" -v 10.13 || exit 1;
+        "$TOOLS/FetchMacOS/fetch.sh" -v 10.13 -k BaseSystem || exit 1;
         ;;
     -m|--mojave)
-        "$TOOLS/FetchMacOS/fetch.sh" -v 10.14 || exit 1;
+        "$TOOLS/FetchMacOS/fetch.sh" -v 10.14 -k BaseSystem || exit 1;
+        ;;
+    -b|--big-sur)
+        "$TOOLS/FetchMacOS/fetch.sh" -v 10.16 -c DeveloperSeed -p 001-18401-003 || exit 1;
```

Comment:
`-v 11`
https://swscan.apple.com/content/catalogs/others/index-11-10.15-10.14-10.13-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog

Reply:
I don't think that using a **DeveloperSeed** is a good idea now that Big Sur is officially available.